book/history/sourdough-history.tex

```diff
@@ -21,17 +21,6 @@ \chapter{The history of sourdough}%
 the evening of December~31.  It seems that humans managed to arrive just
 in time to join the celebration at the end of the year.
 
-The story of sourdough bread begins in ancient oceans. These oceans were the
```

Comment:
Not 100% sure that's the section that should be removed, I tried to infer from the look of it. If you can confirm @hendricius